third_party/closure/goog/caja/string/html/htmlparser.js

```diff
@@ -518,8 +518,10 @@ goog.string.html.HtmlParser.prototype.stripNULs_ = function(s) {
  */
 goog.string.html.HtmlParser.prototype.unescapeEntities_ = function(s) {
   return s.replace(
-      goog.string.html.HtmlParser.ENTITY_RE_,
-      goog.bind(this.lookupEntity_, this));
+      goog.string.html.HtmlParser.ENTITY_RE_, goog.bind(
```

Comment:
@joeltine I force pushed, so it seems your original comment was deleted. But, I agree about the unused parameter.